Possible labile test in test_transerror.py

There seems to be a labile test in test_transerror.py [1]
``git blame`` lists @yarikoptic for code around 312 - ping!

```
======================================================================
FAIL: Test AUC computation
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/PyMVPA/PyMVPA/mvpa2/tests/test_transerror.py", line 312, in test_auc
    self.assertTrue(stats['ACC'] > 1.2 / Nlabels)
AssertionError: 
 Single scenario lead to failures of unittest test_auc:
  on
    clf=<GNB on 5%(ANOVA)> :
     False is not true
    clf=<LinSVM on 5%(ANOVA)> :
     False is not true
```

[1] https://travis-ci.org/PyMVPA/PyMVPA/jobs/122582670

--- a/mvpa2/tests/test_transerror.py
+++ b/mvpa2/tests/test_transerror.py
@@ -309,7 +309,7 @@ class ErrorsTests(unittest.TestCase):
             # its testing labile
             if (('lars' in clf.__tags__) and cfg.getboolean('tests', 'labile', default='yes')) \
                 or (not 'lars' in clf.__tags__):
-                self.assertTrue(stats['ACC'] > 1.2 / Nlabels)
+                self.assertTrue(stats['ACC'] > 1.15 / Nlabels)
             auc = stats['AUC']
             if (Nlabels == 2) or (Nlabels > 2 and auc[0] is not np.nan):
                 mauc = np.min(stats['AUC'])

